UltimateTeam.Toolkit/Requests/LoginRequest.cs

```diff
@@ -162,7 +162,7 @@ private static string GetNucleusPersonaPlatform(Platform platform)
         private async Task<UserAccounts> GetUserAccountsAsync(Platform platform)
         {
             HttpClient.RemoveRequestHeader(NonStandardHttpHeaders.Route);
-            var route = $"https://utas.{(platform == Platform.Xbox360 || platform == Platform.XboxOne ? "external.s3" : "s2")}.fut.ea.com:443";
+            var route = $"https://utas.external.{(platform == Platform.Xbox360 || platform == Platform.XboxOne ? "external.s3" : "s2")}.fut.ea.com:443";
```

Comment:
I guess it's likely that `external.s3` should be changed to just `s3` as well? As it is it will be `https://utas.external.external.s3.fut.ea.com:443` for Xbox.